packages-exp/auth-exp/demo/public/index.html

```diff
@@ -1,763 +0,0 @@
-<!DOCTYPE html>
```

Comment:
you're removing the demo?

Reply:
woops, just meant to get rid of the index.js and index.js.map